rpcs3/Emu/Cell/SPURecompiler.cpp

```diff
@@ -15,7 +15,7 @@ spu_recompiler_base::~spu_recompiler_base()
 
 void spu_recompiler_base::enter(SPUThread& spu)
 {
-	if (spu.pc >= 0x40000 || spu.pc % 4)
+	if (spu.pc & 0xfffc0003) 
```

Comment:
Can I ask why?

Reply:
@woj1993 two checks to test bits of the same value , composed them into one

Reply:
saw that while browsing the files to understand how the emu works.

Reply:
I can write ~0x3fffc instead

Reply:
No. I was wondering if you found any error here.

Reply:
at least leave the intent in comments. Even if compiler doesn't optimize this automatically and this micropt. actually helps,  the intent should be clear.